index.js

```diff
@@ -2632,6 +2642,7 @@
 //. [`Number`]:             #Number
 //. [`Object.create`]:      https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/create
 //. [`Pair`]:               #Pair
+//. [`PositiveInteger`]:    #PositiveInteger
```

Comment:
I had removed it, then it came back in the merge. I've already removed it again (although haven't committed/pushed). I'm something of a git newb when it comes to using it collaboratively